resources/path_collector/path_collector_make_condor.py

```diff
@@ -54,6 +55,12 @@ def make_condor_file(  # pylint: disable=R0913,R0914
         chunk_size_arg = f"--chunk-size {chunk_size}" if chunk_size else ""
         fast_forward_arg = "--fast-forward" if fast_forward else ""
 
+        accounting_group_attr = (
+            f'\n+AccountingGroup="{accounting_group}.$ENV(USER)"\n'
```

Comment:
While `$ENV(USER)` generally works, it's probably better to put the user in directly.  You can get that in python via 
```python
import getpass
getpass.getuser()
```

Reply:
Like this `{accounting_group}.{getpass.getuser()}`?

Reply:
Yes, like that.